template/Dockerfile.oracle

```diff
@@ -108,7 +101,21 @@ RUN set -eux; \
 # 5.7 Debian-based images also included "/etc/mysql/mysql.conf.d" so let's include it too
 	{ echo '!includedir /etc/mysql/mysql.conf.d/'; } >> /etc/my.cnf; \
 	mkdir -p /etc/mysql/mysql.conf.d; \
+	\
+# comment out a few problematic configuration values
+	find /etc/my.cnf /etc/mysql/ -name '*.cnf' -print0 \
+		| xargs -0 grep -lZE '^(bind-address|log)' \
+		| xargs -rt -0 sed -Ei 's/^(bind-address|log)/#&/'; \
```

Comment:
These lines are copied from the Debian Dockerfile even though we don't have a `bind-address` in the `cnf` file.